### PR TITLE
Default layer definitions

### PR DESCRIPTION
**Pull request by SchrodingersGat**
This PR adds default layer definitions for the different types of pads
- Added default layers for various pad types
- Added default pin number

### CODE REVIEW

File: KicadModTree/nodes/base/Pad.py [hunk 1]
```diff
@@ -32,6 +32,10 @@ class Pad(Node):
     SHAPE_RECT = 'rect'
     SHAPE_TRAPEZE = 'trapezoid'
     _SHAPES = [SHAPE_CIRCLE, SHAPE_OVAL, SHAPE_RECT, SHAPE_TRAPEZE]
+    
+    LAYERS_SMT = ['F.Cu','F.Mask','F.Paste']
+    LAYERS_THT = ['*.Cu','*.Mask']
+    LAYERS_NPTH = ['*.Cu']
```
Comment (pointhi) [inline KicadModTree/nodes/base/Pad.py]: why no ```'*.Mask'``` for NPTH pads?

**Comment by pointhi** [comment]
please fix pep8:

```
/home/travis/build/pointhi/kicad-footprint-generator/KicadModTree/nodes/base/Pad.py:35:1: W293 blank line contains whitespace
/home/travis/build/pointhi/kicad-footprint-generator/KicadModTree/nodes/base/Pad.py:36:25: E231 missing whitespace after ','
/home/travis/build/pointhi/kicad-footprint-generator/KicadModTree/nodes/base/Pad.py:36:34: E231 missing whitespace after ','
/home/travis/build/pointhi/kicad-footprint-generator/KicadModTree/nodes/base/Pad.py:37:25: E231 missing whitespace after ','
/home/travis/build/pointhi/kicad-footprint-generator/KicadModTree/nodes/base/Pad.py:54:42: E231 missing whitespace after ','
/home/travis/build/pointhi/kicad-footprint-generator/KicadModTree/nodes/base/Pad.py:54:48: E261 at least two spaces before inline comment
/home/travis/build/pointhi/kicad-footprint-generator/KicadModTree/nodes/base/Pad.py:54:49: E262 inline comment should start with '# '
/home/travis/build/pointhi/kicad-footprint-generator/KicadModTree/nodes/base/Pad.py:113:5: E265 block comment should start with '# '
```

File: KicadModTree/nodes/base/Pad.py [hunk 2]
```diff
@@ -47,7 +51,7 @@ def __init__(self, **kwargs):
         self._initLayers(**kwargs)
 
     def _initNumber(self, **kwargs):
-        self.number = kwargs.get('number')
+        self.number = kwargs.get('number', '""')  # default to an un-numbered pad
```
Comment (pointhi) [inline KicadModTree/nodes/base/Pad.py]: hm, I think we should manage such things inside KicadFileWriter

Comment (SchrodingersGat) [inline KicadModTree/nodes/base/Pad.py]: @pointhi if you think this is the better idea, where should this code go?

I like to default to an "un-numbered" pad for e.g. mechanical mounting pads

**Comment by SchrodingersGat** [comment]
@pointhi if you can help me address the issue you have raised above, I'd like to see this merged :)

File: KicadModTree/nodes/base/Pad.py (hunk 2 above)
Comment (pointhi) [inline KicadModTree/nodes/base/Pad.py]: I think those cases should be handled in KicadFileHandler

**Comment by pointhi** [comment]
I think it's fine for now. I improve the number name handling later on

**Pull request merged**